src/mjolnir/graphvalidator.cc

```diff
@@ -631,7 +631,7 @@ void GraphValidator::Validate(const boost::property_tree::ptree& pt) {
       }
       sum += density;
     }
-    float average_density = sum / densities[level].size();
+    float average_density = densities[level].size() ? sum / densities[level].size() : 0.f;
```

Comment:
For `astar` test only for level 2 `densities[level].size()` is non-zero. Can it be because of test map is not "realistic" or this is an expected result for some input data?